model/oauth/client.go

```diff
@@ -891,3 +899,144 @@ func PushClientsLimitAlert(i *instance.Instance, clientName string, clientsLimit
 }
 
 var _ couchdb.Doc = &Client{}
+
+// OpenIDConfiguration represents the OpenID Connect configuration from the well-known endpoint
+type OpenIDConfiguration struct {
```

Comment:
What do you think about naming this type `OIDCConfiguration` instead? This way we use OIDC everywhere which makes it easier to search for.